src/domain/collaboration/callout/callout.service.ts

```diff
@@ -364,4 +370,16 @@ export class CalloutService {
 
     return aspect;
   }
+
+  public async getComments(calloutId: string) {
```

Comment:
Why is this done via direct repository access instead of going through ORM layer? We only tend to use repository access where performance is critical or the relationships are not navigatable. 

So I would prefer to see using getCalloutOrFail + then passing in a relation to ensure that the comments are loaded (am presuming they have eager loaded = false).